docs_src/getting-started/Ch-GettingStartedHybrid.md

```diff
@@ -1,156 +1,80 @@
 # Working in a Hybrid Environment
 
-In some cases, as a developer, you want to work on a particular micro
-service, but you don't want to have to download all the source code for
-all the micro services and run all the micro services in the development
-tool(s) like Go Land. In this case, you can download and run the EdgeX
-Docker containers for all the micro services you need and run your
-single micro service (the one you are presumably working on) from the
-developer tool of choice and have it point to the other micro services
-by appropriate address. Within EdgeX, we call this a "hybrid"
-environment - where part of your EdgeX platform is running from a
-development environment, while other parts are running from the
-Dockerized containers. This page outlines how to do hybrid development.
-
-As an example of this process, let's say you want to do coding work
-with/on the Virtual Device service in Go Land. You want the rest of the
-EdgeX environment up and running via Docker containers. How would you
-set up this hybrid environment? Let's take a look.
+In some cases, as a [developer or contributor](../general/Definitions.md#contributordeveloper), you want to work on a particular micro service. Yet, you don't want to have to download all the source code, and then build and run for all the micro services. In this case, you can download and run the EdgeX Docker containers for all the micro services you need and run your single micro service (the one you are presumably working on) natively or from a developer tool of choice outside of a container. Within EdgeX, we call this a "hybrid" environment - where part of your EdgeX platform is running from a development environment, while other parts are running from the Dockerized containers. This page outlines how to do hybrid development.
+
+As an example of this process, let's say you want to do coding work with/on the Virtual Device service. You want the rest of the
+EdgeX environment up and running via Docker containers. How would you set up this hybrid environment? Let's take a look.
 
 ## Get and Run the EdgeX Docker Containers
 
 1.  Per [Getting Started Users](./Ch-GettingStartedUsers.md), get
```

Comment:
How about: 
 1. If you haven't already, follow the [Getting Started with Docker](./Ch-GettingStartedUsers.md) Guide before continuing.